default/python/universe_generation/galaxy.py

```diff
@@ -109,15 +422,13 @@ def spiral_galaxy_calc_positions(positions, arms, size, width):
             break
 
         if not attempts:
-            print "Spiral galaxy shape: giving up on placing star", i,\
-                  ", can't find position sufficiently far from other systems"
+            print "Spiral galaxy shape: giving up on placing star {}, can't find position sufficiently far from other systems.".format(i)
```

Comment:
[72/79](https://www.python.org/dev/peps/pep-0008/#maximum-line-length), optionally expanded to 72/99. It was mentioned to me [previously](https://github.com/freeorion/freeorion/pull/567#issuecomment-202140779) to expand to 120.  The wiki entry could use a mention of this if 120 should be the guideline.